package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "scripts": {
+    "ci:set-env": "zx ./scripts/ci/set-env.mjs",
+    "programs:build": "zx ./scripts/program/build.mjs",
+    "programs:test": "zx ./scripts/program/test.mjs",
+    "programs:clean": "zx ./scripts/program/clean.mjs",
+    "programs:format": "zx ./scripts/program/format.mjs",
+    "programs:lint": "zx ./scripts/program/lint.mjs",
+    "generate:idls": "zx ./scripts/generate-idls.mjs",
+    "generate": "pnpm generate:idls && pnpm generate:clients",
+    "generate:clients": "zx ./scripts/generate-clients.mjs",
+    "validator:start": "zx ./scripts/start-validator.mjs",
+    "validator:restart": "pnpm validator:start --restart",
+    "validator:stop": "zx ./scripts/stop-validator.mjs",
+    "clients:js:lint": "zx ./scripts/client/lint-js.mjs",
+    "clients:js:test": "zx ./scripts/client/test-js.mjs",
+    "clients:rust:lint": "zx ./scripts/client/lint-rust.mjs",
```

Comment:
We should have the same options as the program, `*:format`, `*:clippy` and `*:lint`.

Reply:
Same thing, https://github.com/solana-program/feature-gate/pull/8#discussion_r1661326060